release-1.8/release_team.md

```diff
@@ -0,0 +1,12 @@
+|  **Role** | **Name** (**GitHub/Slack ID**)  | **Shadow Name(s) (GitHub/Slack ID), ...**
```

Comment:
Is the "Shadow role" still actual? For 1.7 we had only Lead release manager shadowed.